test/integration/targets/ansible-galaxy-collection-scm/tasks/multi_collection_repo_individual.yml

```diff
@@ -1,5 +1,14 @@
+- name: create requirements file (necessary test an int requirement version)
+  copy:
+    dest: "{{ galaxy_dir }}/requirement_with_int_version.yml"
+    content: |
+      collections:
+        - name: file://{{ test_repo_path }}/.git#collection_2
+          type: git
+          version: 12345
```

Comment:
Could you add a test with an integer version for a non-virtual collection? I wonder if it'll cause a similar traceback.

Reply:
There are some existing tests for non-virtual invalid versions in test/integration/targets/ansible-galaxy-collection-scm/tasks/test_invalid_version.yml. It's in the scm test suite, but it's the virtual dependency (i.e. `type: dir`) with the invalid version.

Reply:
I was initially fixing the unhandled traceback here (by avoiding the cause), but I've changed it to be an error instead and moved the test case into the file with all the other invalid version tests - scm versions need to be quoted.